test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/HeaderCollectionTests.cs

```diff
@@ -73,7 +73,7 @@ public void RequestMessageShouldBeCaseInsensitiveForContentType()
         [Fact]
         public void RequestMessageShouldBeCaseInsensitiveForAccept()
         {
-            TestRequestHeaderRoundTrip("accept", "ACCEPT", "some value");
+            TestRequestHeaderRoundTrip("accept", "ACCEPT", "some/value");

```

Comment:
has the encoding changed

Reply:
@marabooy  passing accept header.. is usually something like application/json not application json